doc/manual/manual.tex

```diff
@@ -1486,6 +1486,85 @@ \subsubsection{Free surface stabilization}
+allow flow through them).  That is to say, the solver may be able to come up with a solution to 
+the Stokes operator, but that solution plus an arbitrary rotation is also an equally valid solution.
+
+Another example is a model in a Cartesian box with periodic boundary conditions in the x-direction, 
+and free slip boundaries on the top and bottom. This setup has arbitrary translations along the x-axis 
+in its nullspace, so any solution plus an arbitrary x-translation is also a solution.
+
+A solution with some small power in these nullspace modes should not affect the physics of the simulation. 
+However, the timestepping of the model is based on evaluating the maximum velocities in the solution, 
+and having unnecessary motions can severely shorten the time steps that \aspect{} takes. 
+Furthermore, rigid body motions can make postprocessing calculations and visualization more 
+difficult to interpret.  
+
+\aspect{} allows the user to specify if their model has a nullspace. If so, any power in the nullspace 
+is calculated and removed from the solution after every timestep.
+There are two varieties of nullspace removal implemented: removing net linear/angular momentum, and 
+removing net translations/rotations. 
+
+For removing linear momentum we search for a constant velocity vector $\bf c$ such that 
+
+\begin{equation*}
+\int_\Omega \rho ({\bf u - c}) = 0
+\end{equation*}
+
+This may be solved by realizing that $\int_\Omega \rho {\bf u} = {\bf p}$, the linear momentum, and 
+
+$\int_\Omega \rho = M$, the total mass of the model.  Then we find 
+\begin{equation*}
+{\bf c} = {\bf p}/M
+\end{equation*}
+
+which is subtracted off of the velocity solution.
+ 
+Removing the angular momentum is similar, though a bit more complicated. 
+We search for a rotation vector $\mathbf \omega$ such that 
+
+\begin{equation*}
+\int_\Omega \rho ( {\bf x \times (u - {\mathbf \omega} \times x) } ) = 0
+\end{equation*}
+
+Recognizing that $\int_\Omega \rho {\bf x \times u} = {\bf H}$, the angular momentum, 
+and $\int_\Omega \rho {\bf x \times {\mathbf \omega} \times x} = {\bf I \cdot {\mathbf \omega} }$, 
+the moment of inertia dotted into the sought-after vector, we can solve for ${\mathbf \omega}$: 
+\begin{equation*}
+{\mathbf \omega} = {\bf I^{-1} \cdot H}
+\end{equation*}
+
+A rotation about the rotation vector $\omega$ is then subtracted from the velocity solution.
+
+Removing the net translations/rotations are idential to their momentum counterparts, but for those the 
+density is dropped from the formulae. For most applications the density should not vary so wildly 
+that there will be an appreciable difference between the two varieties, 
+though removing linear/angular momentum is more physically motivated.
+
+Figure \ref{fig:rigid_rotation} shows the result of removing angular momentum from a convection 
+model in a 2D annulus with free-slip velocity boundary conditions. 
```

Comment:
It would be useful to identify here what the parameter is that the user needs to set.

Other than that, I like this text a lot -- many thanks!